msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md

```diff
@@ -40,31 +40,30 @@ To use these permissions, you must add a [webApplicationInfo](../../resources/sc
 > [!div class="checklist"]
 > [!div class="checklist"]
 >
-> * **id**  — your Azure AD app id.
+> * **id**  — your Azure AD app ID.
 > * **resource** — the resource URL for the app.
 >
-
 >[!NOTE]
 >
-> * Your bot requires _application_ not _user delegated_ permissions because the installation is not for yourself but for others.
+> * Your bot requires application and not user delegated permissions because the installation is for others.
 >
-> * An Azure AD tenant administrator must [explicitly grant permissions to an application](/graph/security-authorization#grant-permissions-to-an-application). After an application is granted permissions, _all_ members of the Azure AD tenant will gain the granted permissions.
+> * An Azure AD tenant administrator must [explicitly grant permissions to an application](/graph/security-authorization#grant-permissions-to-an-application). After an application is granted permissions, _all_ members of the Azure AD tenant gains the granted permissions.
 
 ## Enable proactive app installation and messaging
 
  > [!IMPORTANT]
->Microsoft Graph will only install apps published within your organization's [app catalog](../../concepts/deploy-and-publish/overview.md#publish-to-your-organizations-app-catalog) or in [AppSource](https://appsource.microsoft.com/).
+>Microsoft Graph can only install apps published within your organization's [app catalog](../../concepts/deploy-and-publish/overview.md#publish-to-your-organizations-app-catalog) or in [AppSource](https://appsource.microsoft.com/).
 
 ### ✔ Create and publish your proactive messaging bot for Teams
 
-To get started, you will need a [bot for Teams](../../bots/how-to/create-a-bot-for-teams.md) with [proactive messaging](../../concepts/bots/bot-conversations/bots-conv-proactive.md) capabilities and  [published](../../concepts/deploy-and-publish/overview.md) in your organization's [app catalog](../../concepts/deploy-and-publish/overview.md#publish-to-your-organizations-app-catalog) or in [AppSource](https://appsource.microsoft.com/).
+To get started, you must need a [bot for Teams](../../bots/how-to/create-a-bot-for-teams.md) with [proactive messaging](../../concepts/bots/bot-conversations/bots-conv-proactive.md) capabilities and  [published](../../concepts/deploy-and-publish/overview.md) in your organization's [app catalog](../../concepts/deploy-and-publish/overview.md#publish-to-your-organizations-app-catalog) or in [AppSource](https://appsource.microsoft.com/).
 
 >[!TIP]
 > The production-ready [**Company Communicator**](../..//samples/app-templates.md#company-communicator) app template enables broadcast messaging and is a good foundation for building your proactive bot application.
 
 ### ✔ Get the `teamsAppId` for your app
 
-**1.** You will need the `teamsAppId`  for the next steps.
+**1.** You must need the `teamsAppId` for the next steps.
```

Comment:
Please check the usage of **must** throughout the document, wherever an update has been made.